deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1106,17 +1093,15 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
     }
 
     private func reloadData() {
-        let selectredRows = tableView.indexPathsForSelectedRows
         tableView.reloadData()
         // There's an iOS bug, filling up the console output but which can be ignored: https://developer.apple.com/forums/thread/668295
         // [Assert] Attempted to call -cellForRowAtIndexPath: on the table view while it was in the process of updating its visible cells, which is not allowed.
-        selectredRows?.forEach({ (selectedRow) in
+        tableView.indexPathsForSelectedRows?.forEach({ (selectedRow) in
```

Comment:
this introduces an unwanted logic change bug as selected rows may change on tableView.reloadData(); https://github.com/deltachat/deltachat-ios/commit/0d29044ff8fdd2e657a5cc664a5164b92545de6f restores the original behaviour